test-collateral/generate-policy/src/main.rs

```diff
@@ -169,6 +172,7 @@ impl Arguments {
             enclave_debug_mode: false,
             execution_strategy: String::new(),
             enable_clock: false,
+            max_memory: 0,
```

Comment:
Would we rather set a rational default value here?
Also, (this is a question up for discussion, not a demand): Do we want to put the units in the field name:
```
max_memory_mb
```
or something or other?

Reply:
> Would we rather set a rational default value here?

Hmm like a fraction?

Reply:
ha ha!

Like "256" - which is what we currently use.

Reply:
Haha got it :+1: